src/search/result.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { View, Text, Image } from 'react-native';
+import { TouchableHighlight } from 'react-native-gesture-handler';
+import styles from './styles';
+
+import tradeIcon from '../../assets/icons/search-page/trade.png';
+import detailIcon from '../../assets/icons/search-page/detail.png';
+
+export default () => {
+  const isLoading = useSelector((state) => state.search.isLoading);
+  const searchResult = useSelector((state) => state.search.result);
+  const error = useSelector((state) => state.search.error);
+
+  if (isLoading) {
+    return <Text>Loading</Text>;
```

Comment:
Please use a [ActivityIndicator](https://facebook.github.io/react-native/docs/activityindicator) to handle the loading state
```suggestion
    return <ActivityIndicator size="large" />;
```